pkg/controller/controller.go

```diff
@@ -43,8 +45,14 @@ func init() {
 }
 
 func getLogger(opts ...zap.Opts) logr.Logger {
+	// make raw logger object from options in original zap pkg
+	rawOpts := zap.RawZapOpts(ubzap.AddCaller())
```

Comment:
why can we not pass this in to the controlleruntime zap.New?

Reply:
no such option exists in the package. tried using a new [json encoderconfig](https://github.com/Azure/aks-app-routing-operator/pull/72/files#diff-243ebed2765f75e6a54f57167212fefb08c3b2a85967ad2acbc0eb78919019c1R28) with the caller key field populated but that didn't work

Reply:
<img width="883" alt="image" src="https://github.com/Azure/aks-app-routing-operator/assets/33605867/8fd2ca72-efcd-4698-8958-a660939627eb">

Reply:
No need for rawLogger or zapr